internal/log/log.go

```diff
@@ -36,14 +37,14 @@ const (
 	PanicSeverity
 )
 
-// severityLevels is a list of all available severity levels. This is a shorthand
-// to work around the fact that Golang cannot `range` over an enum.
-var severityLevels = []severity{
-	DebugSeverity,
-	InfoSeverity,
-	WarnSeverity,
-	ErrorSeverity,
-	PanicSeverity,
+// severityLevels is a mapping of all available severity levels to their printed
+// values.
+var severityLevels = map[severity]string{
+	DebugSeverity: "DEBUG",
+	InfoSeverity:  "INFO",
+	WarnSeverity:  "WARN",
+	ErrorSeverity: "ERROR",
+	PanicSeverity: "PANIC",
```

Comment:
Minorish thing: Not directly related to this PR, but I noticed with doing the panic fix that the `Severity` suffix on the constant names didn't add anything and felt like it was actually harming readability.  I say we just drop it.